.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
+
+
+
+**What is the current behavior? (You can also link to an open issue here)**
+
+
+
+**What is the new behavior (if this is a feature change)?**
+
+
+
+**Does this PR introduce a breaking change?**
+
+
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit follows our guidelines: https://github.com/serilog/serilog#contribute
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
```

Comment:
Currently we rely on the wiki for our docs.  This is a great check however not sure we could enforce ATM